src/frontends/common/include/openvino/frontend/frontend.hpp

```diff
@@ -83,9 +83,10 @@ class FRONTEND_API FrontEnd {
     virtual void convert(const std::shared_ptr<ov::Model>& partially_converted) const;
 
     /// \brief Convert only those parts of the model that can be converted leaving others
-    /// as-is. Converted parts are not normalized by additional transformations; normalize
-    /// Model or another form of convert Model should be called to finalize the
-    /// conversion process.
+    /// as-is wrapped by FrameworkNode. Converted parts are normalized by additional
+    /// transformations like it is done in convert method. If part of the graph cannot be
+    /// converted, it is not guaranteed that the converted regions are complitelly normalized.
```

Comment:
```suggestion
    /// converted, it is not guaranteed that the converted regions are completely normalized.
```

Reply:
what does it mean "Converted parts" here? 
if it means that we are going to run "normalize" transformations for the part which were wrapped by FrameworkNodes, then we can't mention "like it is done in convert method" here.  Because, for example, "convert" method of TF FE doesn't call  "normalize" for them. Should we fix "convert" method in TF FE?

It's quite strange to see that
auto input_model = fe->load(path);
fe->convert(input_model) // fails
fe->convert_partially(input_model) // works well and the model fully normalized

Reply:
@itikhono, `fe->convert_partially(input_model)` will give you a partially converted model. The parts of the model that consist of converted operations ("converted parts") will be normalized by regular normalization passed. But as the model cannot be converted completely then it is not "fully normalized" in your terminology, because normalization passes don't know how to deal with unknown FrameworkNodes and it may lead to under-normalizations. For example, your love -- transpose sinking -- cannot transfer Transpose over FrameworkNode, because the semantics of this node is unknown. FrameworkNode cannot produce output shape and element_type, then it may break some transformations that they won't do their job for the entire model.

It already works in this way.

Reply:
in the case above, 
auto input_model = fe->load(path);
fe->convert(input_model) // fails because there are some nodes which are not supported for 1:1 fw -> ov mapping, but can be normalized
fe->convert_partially(input_model) // works well and the model fully normalized
after "convert_partially" method call we will get the model without Framework nodes, these nodes will be converted to TensorIterator and so on, but "convert" method doesn't run the part of transformations in TF FE, which can help to get rid of FrameworkNodes